src/main/java/appeng/recipes/loader/RecipeResourceCopier.java

```diff
@@ -225,9 +223,9 @@ private void copyFile( @Nonnull final File destination, @Nonnull final String di
 			{
 				/* A JAR path */
 				final String dirPath = dirURL.getPath();
-				final String jarPath = dirPath.substring( 5, dirPath.indexOf( '!' ) ); // strip out only
+				final String jarPath = dirPath.substring( 0, dirPath.indexOf( '!' ) ); // strip out only
 				// the JAR file
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS106)